datetime/src/main/java/com/afollestad/materialdialogs/datetime/utils/ViewExt.kt

```diff
@@ -0,0 +1,37 @@
+/**
+ * Designed and developed by Aidan Follestad (@afollestad)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.afollestad.materialdialogs.datetime.utils
+
+import android.os.Build
+import android.widget.TimePicker
+
+fun isNougat() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
+
+@Suppress("DEPRECATION")
```

Comment:
Rather than having this suppression on each method, it can be at the top of the file (IntelliJ has an option for that when you do Alt+Enter)